parsely/src/main/java/com/parsely/parselyandroid/ParselyMetadata.java

```diff
@@ -18,7 +17,7 @@
 public class ParselyMetadata {
     ArrayList<String> authors, tags;
     String link, section, thumbUrl, title;
-    Calendar pubDate;
+    long publicationDateMilliseconds;
```

Comment:
I'd appreciate feedback on this change. I don't like introducing a breaking change to the library, but at the same time, I would rather not force users to use `java.util.Calendar` API - there's no need to attach to this specific class to share a moment in time.

I've decided to propose a simple, API-agnostic, straightforward and universal `long` property. The consequence of this is that user can now easily provide an incorrect value (e.g. timestamp in seconds), but I sense that it's not that likely (name of the property) and simpler API has more value. WDYT?

Reply:
Could you elaborate on why using `Calendar` would be undesirable - besides the fact that it may not be the particular API the clients are using?

Reply:
The concern about different APIs used by the clients is the main one, but `Calendar` also feels heavier than what we need from this field, which is passing a timestamp.

But, as the majority of the usage is probably passing `Calendar.getInstance()` here, maybe it's not a big deal and backward compatibility and safer API design is better? I'm not sure now.

Reply:
In my opinion, in most cases using _a_ type is better than not having a type at all. One obvious exception to that is when using a specific type requires the clients to add a new dependency. However, I believe `java.util.Calendar` is available without any external dependencies, so I don't see any issues about that.

Even if the type we choose is not exactly the type the clients would like to use, they can always convert to it on a need to basis as long as it is properly documented. In this case, `Calendar` is a well known and properly documented class, so I think it's a very good candidate to be used in external facing APIs.

Reply:
You're right - `Calendar` is a well-known and available dependency for every client. I agree it's easy to convert from any other API. Thank you for sharing your thoughts! - I brought back `Calendar` in 67a755b

Reply:
Thank you! 🙇‍♂️